### PR TITLE
[9.0][ADD] sale_open_qty

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Sale Open Qty
=============

This module allows sale users to identify what are the sale orders
that currently have quantities pending to invoice or to deliver.

Usage
=====

To use this module, you need to go to Sale Orders and select the new filters in the search.

### CODE REVIEW

File: sale_open_qty/README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
```
Comment (LoisRForgeFlow) [inline sale_open_qty/README.rst]: :pensive:

File: sale_open_qty/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        return res
+
+    qty_to_invoice = fields.Float(compute='_compute_qty_to_invoice',
+                                  search='_search_qty_to_invoice',
+                                  string="Qty to Bill",
+                                  default=0.0)
+    qty_to_deliver = fields.Float(compute='_compute_qty_to_deliver',
+                                  search='_search_qty_to_deliver',
+                                  string="Qty to Deliver",
+                                  default=0.0)
```
Comment (houssine78) [inline sale_open_qty/models/sale_order.py:71]: I'm quiet confused about the relevantness of a field that will mainly mix different units of mesure. Having line in kg and pound or g totalized togheter in one field could be quiet confusing. 

Not sure how the field would help the user to follow the open qty that still need to be delivered.

Comment (MiquelRForgeFlow) [reply sale_open_qty/models/sale_order.py:71]: Fixed

**Comment by rousseldenis** [comment]
@MiquelRForgeFlow Is this still applicable ?

**Comment by MiquelRForgeFlow** [comment]
@rousseldenis I don't know if somebody used it.

**Comment by andreampiovesana** [comment]
forever open?

**Comment by rousseldenis** [comment]
> forever open?

We can merge it.

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**